mos-platform/cx16/_6522.h

```diff
@@ -0,0 +1,73 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I just finished relicensing the SDK, so we're only accepting contributions made available under the new license. Accordingly, this should read:

```
// Copyright 2022 LLVM-MOS Project
// Licensed under the Apache License, Version 2.0 with LLVM Exceptions.
// See https://github.com/llvm-mos/llvm-mos-sdk/blob/main/LICENSE for license
// information.
```

Reply:
Fixed